src/Microsoft.Cci.Extensions/project.lock.json

```diff
@@ -3,7 +3,7 @@
   "version": 2,
```

Comment:
There where two indirect dependencies that had very old versions, which existed in dotnet-corefx feed but they were not available in dotnet-core one. The restore was failing because those two dependencies couldn't be found, so I just updated the lock file and that fixed it, by upgrading the versions of both of those to be able to find them in dotnet-core feed.